SUPPORT.md

```diff
@@ -0,0 +1,38 @@
+## Kubernetes User Support
```

Comment:
Perhaps change this to be more friendly.. like "Support for deploying and using Kubernetes"